src/workflows/airqo_etl_utils/data_api.py

```
@@ -0,0 +1,908 @@
+            "nowcastonly": 1,
+            "includerawconcentrations": 1,
+            "dataType": "B",
+        }
+
+        params.update(integration.get("auth", None))
+        return self.__request(endpoint, params, network=DeviceNetwork.METONE)
+
+    def __request(
+        self,
+        endpoint: str,
+        params: Optional[Dict[str, Any]] = {},
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)